invokeai/backend/model_manager/util/select_hf_files.py

```diff
@@ -71,7 +79,8 @@ def filter_files(
         paths = [x for x in paths if x.parent == Path(subfolder)]
 
     # _filter_by_variant uniquifies the paths and returns a set
-    return sorted(_filter_by_variant(paths, variant))
+    # jennifer removed the filter since it removed models, probably will break something but i dont understand why it removes valid models :|
```

Comment:
Why are we filtering by ModelRepoVariant.Default? I tested removing this and was able to download and use models that this filtered out

Reply:
It's needed for specific edge cases of HF models. I imagine you didn't hit any of them during your testing